docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -3,104 +3,93 @@ title: Run load-balanced VMs on Azure for scalability and availability
 
-In this architecture, a workload is distributed across several VM instances. There is a single public IP address, and Internet traffic is distributed to the VMs using a load balancer. This architecture can be used for a single-tier application, such as a stateless web application or storage cluster. It's also a building block for N-tier applications. 
+In this architecture, a workload is distributed across several VM instances. There is a single public IP address, and Internet traffic is distributed to the VMs using a load balancer. This architecture can be used for a single-tier application, such as a stateless web application.
 
-The architecture has the following components: 
+The architecture has the following components:
 
-* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set. 
+* **Resource group.** A [*resource group*][resource-manager-overview] is a container that holds related resources. You usually create resource groups for different resources in a solution based on their lifetime, and who will manage the resources.
 * **Virtual network (VNet) and subnet.** Every VM in Azure is deployed into a VNet that is further divided into subnets.
 * **Azure Load Balancer**. The [load balancer] distributes incoming Internet requests to the VM instances. The load balancer includes some related resources:
   * **Public IP address**. A public IP address is needed for the load balancer to receive Internet traffic.
   * **Front-end configuration**. Associates the public IP address with the load balancer.
   * **Back-end address pool**. Contains the network interfaces (NICs) for the VMs that will receive the incoming traffic.
-* **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool. 
-* **Network address translation (NAT) rules**. Used to route traffic to a specific VM. For example, to enable remote desktop protocol (RDP) to the VMs, create a separate NAT rule for each VM. 
-* **Network interfaces (NICs)**. Each VM has a NIC to connect to the network.
+* **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
+* **VM Scaleset**. A [VM scaleset][vm-scaleset] contains a set of VMs with identical settings, used to host the same workload. Scaelsets allow to scale the number of VMs out and in manually, or automatically based on predefined rules.
+* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set.
```

Comment:
The availability set contains the VMs, making the VMs highly available and covering them with the availability set service level agreement. In order for the SLA to apply, the availability set must include a minimum of two VMs.

Reply:
Remove availability set from this list and from the diagram. VMSS is "implicitly" an availability set (per the product docs) but it's not deployed as a separate resource.

Reply:
I gave this some thought. I will keep the availability set and add to the fact that it is implicit. The avset resource still exists.

Reply:
Ok, I will keep the avset text, but add to it that it is implicit. Thanks Mike!